_includes/header.html

```diff
@@ -1,17 +1,20 @@
 <nav class="navbar navbar-expand-md noprint{% if site.navbar_sticky %} sticky-top{% endif %}">
 
   <a class="navbar-brand" href="/">
-    {% include opensuse.svg %}
+    <img src="https://static.opensuse.org/favicon.svg" class="d-inline-block align-top" alt="🦎" title="openSUSE"
```

Comment:
Alt for our logo should always be `openSUSE`, this is what it semantically means